src/pages/EachArticle.jsx

```diff
@@ -6,13 +6,17 @@ import { getArticleByID } from "../utils/api";
 const EachArticle = () => {
   const { article_id } = useParams();
   const [article, setArticle] = useState({});
+  const [isLoading, setIsLoading] = useState(true);
 
   useEffect(() => {
     getArticleByID(article_id).then(({ article }) => {
       setArticle(article);
+      setIsLoading(false);
     });
   }, [article_id]);
 
+  if (isLoading) return <p className="loadingMsg">Fetching Article...</p>;
+
```

Comment:
Just needs some handling of id's that don't exist. At the moment, if i change the url to one that isnt there, i just get a constant loading page